x-pack/plugins/fleet/common/constants/routes.ts

```diff
@@ -171,6 +171,7 @@ export const ENROLLMENT_API_KEY_ROUTES = {
 
 export const UNINSTALL_TOKEN_ROUTES = {
   LIST_PATTERN: `${API_ROOT}/uninstall_tokens`,
+  INFO_PATTERN: `${API_ROOT}/uninstall_tokens/{agentPolicyId}`,
```

Comment:
Should we have a route like `/agent_policies/{agentPolicyId}/uninstall_tokens`? 

In my opinion the route `/uninstall_tokens/{id}` (if we need it) should return one specific uninstall token

Reply:
yeah, I think you're right! the routes you suggested harmonize better with rest api conventions.

@dasansol92 what do you think? we could have 3 routes:
- the original `/uninstall_tokens?policy_id=xxx` used by the table, but with adding the uninstall tokens' `id`s to the response: `{ id, policy_id, created_at }`
- we could rename the second route (line 174) to the suggested `/agent_policies/{agentPolicyId}/uninstall_tokens/` route for the token history (will be used by the flyout)
- we could add a new endpoint `/uninstall_tokens/{id}` that can be used in the uninstall token table, when the user presses the 👁️ button, to fetch only that token (so instead of fetching the token history and selecting the top element)

Reply:
That sounds good to me, using the `id` makes things easier. Note that in this case the flyout could be using the first one or the third one (depending on how we want to handle it). If from the table, using the third one (as we could set the uninstall token id as a flyout prop), else (from policies list, agents list, etc) using a combination of first + third. I would go for something simple for the first iteration and use first one for policy search and third one for get uninstall token. 

With that path, the flyout should perform two API calls when outside the table, one to get all the policyId/uninstall tokens, get the last created_at from the response and then get the token by id with the third API. Thoughts?

Reply:
good thoughts! so you think that (2) should not decrypt the token history? I assumed that it should return it decrypted, and in this case, we only need one API call in the flyout: it can use this (2) route to fetch the decrypted token history and show the latest of them (for now). but if you think it's better to decrypt only with (3), I'm good with that, too, no strong opinion on this. what do you think?

Reply:
I think in the current version it's okay to decrypt, because there will be only one token per policy. some months in, when we know how the token history feature should look like or how the Uninstall Command Flyout changes, we can adapt, e.g. by adding a new route (`GET /agent_policy/{agentPolicyId}/latest_uninstall_token`) if needed, but for now, I think we're good.

Reply:
this route is removed, so now we only have two routes: to get multiple token _metadata_ and to get _one_ decrypted token